### DISCUSSION TITLE
How to color an SVG using URL parameters?

### DISCUSSION THREAD

**Discussion by f3liiix** [tailwindlabs/tailwindcss#14012] [Help]
like this: [https://tailwindui.com/img/logos/mark.svg?color=lime&shade=500](https://tailwindui.com/img/logos/mark.svg?color=lime&shade=500)

**Comment by wongjn** [comment]
You could consider using a [service worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) or some website server logic to intercept the file content, rewriting the file content as appropriate for the URL parameters before it gets delivered to the browser.

**Reply by f3liiix** [reply]
Thank you, are there any related plugins or tutorials that I can refer to?

**Reply by wongjn** [reply]
You could try searching the internet for those resources
